packages/grid/_modules_/grid/utils/exportAs.ts

```diff
@@ -0,0 +1,23 @@
+export const exportAs = (blob: Blob, extension: string, filename: string): void => {
```

Comment:
Assuming the method was copied from the initial PR, I think that keeping the comment/sources of inspiration are important. We can also remove IE 11 related code

Reply:
I took it from elsewhere but yes I can add the comments from the other one.

Reply:
done